extensions/amp-accordion/validator-amp-accordion.protoascii

```diff
@@ -61,6 +61,10 @@ tags: {  # <amp-accordion> > <section>
   spec_name: "amp-accordion > section"
   mandatory_parent: "AMP-ACCORDION"
   attrs: { name: "expanded" value: "" }
+  # amp-bind
+  attrs: {
+    name: "[data-expand]"
```

Comment:
Did you mean data-expanded?

Reply:
The name was originally `[expanded]` but later changed to `[data-expand]`, so the title and description of the PR look to be out of date, but the code looks to be consistent